SU-HSDO/suhumsci
# SHS-2907: Reduce font loading to increase site performance

# READY FOR REVIEW
Clickup Ticket - https://app.clickup.com/t/36718269/SHS-2907

## Summary
- Update decanter to the latest version which contains the `font-display` changes we needed from decanter - https://github.com/SU-SWS/decanter/pull/892 

## Need Review By (Date)
2/27

## Urgency
low

## Steps to Test
1. Delete this existing `node_modules` from `/docroot/themes/humsci/humsci_basic` 
2. Run `npm install` from `/docroot/themes/humsci/humsci_basic`
3. run `npm run build` and verify there are no compiling errors. Note: We can ignore the Decanter 7 warnings.
4. Verify the correct version of decanter was installed and the `font-display` property is present in `/docroot/themes/humsci/humsci_basic/node_modules/decanter/core/src/scss/core/fonts/_fonts.scss`
5. Verify there are no notable changes in the percy tests below.
https://percy.io/da667d23/suhumsci/builds/25326447
https://percy.io/da667d23/suhumsci/builds/25326453


## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
